dask_image.Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG NUMPY_VER=1.20.1
```

Comment:
Should NumPy be pinned here or can we relax a bit ?

Reply:
Yeah let's relax this and defer to the dask-image environment file